presto-orc/src/main/java/com/facebook/presto/orc/metadata/DwrfMetadataWriter.java

```diff
@@ -277,6 +278,8 @@ public int writeStripeFooter(SliceOutput output, StripeFooter footer)
                 return DwrfProto.Stream.Kind.DICTIONARY_COUNT;
             case ROW_INDEX:
                 return DwrfProto.Stream.Kind.ROW_INDEX;
+            case IN_MAP:
```

Comment:
Do not introduce In_Map writer change in this PR. Let us introduce iN_map change when required.